tilt/nvidia-dcgm-daemonset.yaml

```diff
@@ -29,6 +29,8 @@ spec:
       labels:
         app: nvidia-dcgm
     spec:
+      nodeSelector:
+        nvidia.com/gpu.present: "true"
```

Comment:
This will cause the pods to not land on the kind worker right?

Reply:
correct!